### PR TITLE
Separate ZeroEQ communication to class zeq::Communicator, minor cleanups

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: livre/Eq/Client.cpp [hunk 1]
```diff
@@ -149,10 +149,7 @@ int Client::run()
 
     clock.reset();
     while( config->isRunning() && maxFrames-- )
-    {
-        config->startFrame();
-        config->finishFrame();
-    }
+        config->renderFrame();
```
Comment (eile) [inline livre/Eq/Client.cpp]: just config->frame()?

File: livre/Eq/Config.cpp [hunk 2]
```diff
@@ -548,62 +416,7 @@ bool Config::_deregisterFrameData()
...
-                new zeq::Subscriber( lunchbox::URI( subscriberSchema )));
-    _impl->subscribers.push_back( vwsSubscriber );
-    vwsSubscriber->registerHandler( zeq::hbp::EVENT_CAMERA,
-                                    boost::bind( &detail::Config::onCamera,
-                                                 _impl, _1 ));
-    vwsSubscriber->registerHandler( zeq::vocabulary::EVENT_REQUEST,
-                                    boost::bind( &detail::Config::onRequest,
-                                                 _impl, _1 ));
-#  endif
+    _impl->communicator.reset( new zeq::Communicator( *this ));
```
Comment (eile) [inline livre/Eq/Config.cpp]: rm _initZeroEQ and inline again?

**Comment by eile** [comment]
+1

**Comment by tribal-tec** [comment]
Updated

**Comment by eile** [comment]
+1

**Pull request merged**